src/mcp/client/stdio/__init__.py

```diff
@@ -24,6 +24,8 @@
 
 logger = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
```

Comment:
Duplicate lines, can we remove this? Might be because of syncing changes from main into the branch, if so apologies!